contrib/completion/zsh/_docker

```diff
@@ -1083,6 +1083,8 @@ __docker_service_subcommand() {
         "($help)*--constraint=[Placement constraints]:constraint: "
         "($help)--endpoint-mode=[Placement constraints]:mode:(dnsrr vip)"
         "($help)*"{-e=,--env=}"[Set environment variables]:env: "
+        "($help)*--group-add=[Add additional user groups to the container]:group:_groups"
+        "($help)*--group-rm=[Remove previously added user groups from the container]:group:_groups"
```

Comment:
Is this correct? `--group-add` is available for both `docker service update` and `docker service create` but `--group-rm` is only available for `docker service update` (sorry for the misleading subject that I gave to my PR)